keyring/tests/test_backend.py

```diff
@@ -14,7 +14,9 @@
 from .util import random_string
 from keyring import errors
 
-DIFFICULT_CHARS = string.whitespace + string.punctuation
```

Comment:
I moved DIFFICULT_CHARS into BackendBasicTests. '@' will be removed only for KWallet tests.